pilot/test/integration/driver.go

```diff
@@ -85,6 +88,8 @@ func init() {
 		"kube config file (missing or empty file makes the test use in-cluster kube config instead)")
 	flag.IntVar(&count, "count", 1, "Number of times to run the tests after deploying")
 	flag.StringVar(&authmode, "auth", "both", "Enable / disable auth, or test both.")
+	flag.StringVar(&mixermode, "mixer", "enable", "Enable / disable mixer.")
```

Comment:
Mixermode varbool ?

Reply:
what is that? allows you to do enable/disable?